src/mailchimp/mailchimp_manager.py

```diff
@@ -77,9 +95,15 @@ def create_folder(self, foldername: str) -> int:
         Raises:
             TODO: Document all possible errors
         """
-        raise NotImplementedError
-
-    def upload_certificates(self, attendees: AttendeeManager,
+	try:
+                client = MailchimpMarketing.Client()
```

Comment:
The client should just be a protected attribute of the mailchimp manager that is set in the `set_authorisation` method. That way, you don't have to recreate the client object every time `create_folder()` is called